helm/prometheus-operator/templates/deployment.yaml

```diff
@@ -22,6 +22,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
           args:
+            - --kubelet-object={{ .Values.kubeletObject }}
```

Comment:
Yes, probably. I am using the operator to monitor kubernetes itself so I didn't think of that. I will add the conditional.
I am going to enable it by default, unless someone is against it: if the option is here but we don't need it, it's not a big deal.